src/app/_components/Specialists/SpecialistListWithMap.js

```diff
@@ -12,28 +12,138 @@ import { MapLink } from '@components/MapLink';
 import { ShortCardWrapper } from '@components/CardSpecialist/ShortCardWrapper';
 import { getProperEnding } from '@components/Specialists/utils';
 import { NoMatches } from '@components/Specialists/NoMatches';
+import { addressesToPoints } from '@utils/common';
+import { Map } from '@components/Map';
+import { useMediaQuery } from '@mui/material';
 import Loading from '@/app/loading';
+import { screens } from '@/app/styles/tailwind/ui';
+import { useWindowResize } from '@/app/_hooks/useWindowResize';
 
 import 'swiper/css/pagination';
 import 'swiper/css/navigation';
 
+const cardStyle = 'max-w-[900px] rounded-3xl border-2 border-gray-200 px-4 py-5 lg:mx-auto h-full';
+const sliderBreakpoints = {
+  360: {
+    slidesPerView: 1,
+    spaceBetween: 24,
+  },
+  640: {
+    slidesPerView: 1.25,
+    spaceBetween: 16,
+  },
+  768: {
+    slidesPerView: 1.5,
+    spaceBetween: 14,
+  },
+};
+
+const getMappedPointsList = list =>
+  list
+    ?.map(entry => {
+      const isOrganization = entry.organization;
+      const entryData = isOrganization ? entry.organization : entry.specialist;
+      const addressPrimary = entryData.addresses[0];
+
+      if (!addressPrimary) {
+        return [];
+      }
+
+      return addressesToPoints([addressPrimary]).map(address => ({
+        ...address,
+        specialistId: entryData.id,
+      }));
```

Comment:
This can be rewritten in a less verbose way by omitting some of the single-use variables:
```
      const addresses = (entry.organization || entry.specialist).addresses;
      if (!addresses[0]) return [];

      return addressesToPoints(addresses).map(address => ({ ...address, specialistId: entryData.id,
      }));
```

Reply:
It looks better, thanks!